src/app/customer/modals/customer-management-modal/customer-management-modal.component.html

```diff
@@ -23,9 +23,14 @@ <h1 class="modal-title fs-5">Customer</h1>
 
           <div class="col-md-4 w-100">
             <label for="password" class="form-label fw-bold">Password</label>
-            <input type="password" class="form-control" formControlName="password" minlength="8" required="">
+            <div class="input-group">
+                <input [type]="passwordFieldType" class="form-control" formControlName="password" minlength="8" required="">
+                <span class="input-group-text" (click)="togglePasswordVisibility()">
+                    <i class="bi" [ngClass]="{'bi-eye': passwordFieldType === 'password', 'bi-eye-slash': passwordFieldType === 'text'}"></i>
```

Comment:
Convert the NgClass to be a variable and move the condition to typescript please :)